components/automate-ui/src/app/pages/policy/details/policy-details.component.ts

```diff
@@ -70,9 +75,7 @@ export class PolicyDetailsComponent implements OnInit, OnDestroy {
           }
         });
         delete this.policy.members;
-        return members;
-      }));
-    this.members$.subscribe();
```

Comment:
Thank you for cleaning this up. I keep seeing this but thought it was out of the scope of my work.